### ISSUE TITLE
Light Attacks not counted on 'Info' panel

### ISSUE THREAD

**Issue by tkmehta** [Solinur/CombatMetrics#14]
I've been having an issue for a while now where Light Attacks are not counted when viewing the info page. I've disabled all addons to rule out conflicts and the issue still persists. I've also tried the 'Recalculate fight' option to no avail. The 'Fight Stats' panel correctly reports the number of light attacks so I am using this to compare with the 'Info' panel.
![Screenshot from 2022-06-27 13-57-59](https://user-images.githubusercontent.com/10963041/176007871-f6df75b0-1a7c-4fde-9ab9-2b53d02d5760.png)
![Screenshot from 2022-06-27 13-57-54](https://user-images.githubusercontent.com/10963041/176007874-e2d486aa-9f42-4e98-b443-7c1c41a2a997.png)

**Comment by Solinur** [comment]
Hmm thats quite weird. How do you weave? Do you often bar swap cancel or block or bash cancel ?

**Comment by Solinur** [comment]
Yeah the ability tracking in CMX is unique to provide those timing stats.

**Comment by Solinur** [comment]
Can you make a screenshot of the log panel with only skilla activations enabled like in my screenshot?
![Screenshot 2022-06-27 230729](https://user-images.githubusercontent.com/19504443/176036085-a4233d34-53b5-41f1-81a4-f0c194728309.png)

**Comment by Solinur** [comment]
So it looks like it sees dual wield LAs but somehow they are not attributed..

**Comment by tkmehta** [comment]
I scrolled through the log and I didn't see a delay value for light attacks until around 27 sec into the log. 

![image](https://user-images.githubusercontent.com/10963041/176038596-375eaa81-5c27-41d1-9ab2-243bc6c2a84f.png)

**Comment by Solinur** [comment]
Thanks. Not sure how this is possible, but now that I know I should be able to figure out why this is happening.

**Comment by tkmehta** [comment]
Any updates on this? I am still experiencing the issue unfortunately :(

**Comment by Solinur** [comment]
Sorry, unfortunately I haven't been able to figure this out yet. I have to admit though, that I didn't spent as much time on this as I would like to. 
Sorry for your inconvinience, I'll try to do better.

**Comment by tkmehta** [comment]
No need to apologize! I appreciate all your work with this addon. If there's anything I can do to help debug, don't hesitate to ask!

**Comment by Solinur** [comment]
closed for follow-up issue: #21
